web/packages/teleterm/src/ui/services/workspacesService/documentsService/connectToApp.ts

```diff
@@ -111,6 +143,46 @@ export async function connectToApp(
   }
 }
 
+export async function connectToAppWithVnet(
+  ctx: IAppContext,
+  launchVnet: () => Promise<[void, Error]>,
+  target: App
+) {
+  const cluster = ctx.clustersService.findClusterByResource(target.uri);
+
+  const [, err] = await launchVnet();
+  if (err) {
+    return;
```

Comment:
I have an app with an invalid address:
<img width="1117" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/082fc264-f473-46d9-82bc-4e0f4dc9e9d8">

Shouldn't we show the errors somehow? An error notification would be nice, now it fails silently.